scipy/special/tests/test_gammainc.py

```diff
@@ -111,6 +123,14 @@ def test_infinite_limits(self):
             rtol=0
         )
 
+    def test_nan_a_and_x_0(self):
```

Comment:
Same comment as above.

Reply:
same response as above, no need to test twice :)